mozci/task.py

```diff
@@ -68,6 +68,9 @@ def get_artifact(self, path):
 
 @dataclass
 class TestTask(Task):
+    result_group: str = ""
+    result_test: str = ""
+    result_ok: bool = True
```

Comment:
Do you think it would be worth creating a `Result` dataclass to store individual results? Or are just adding unnecessary overhead at that point.

Reply:
Done.